packages/components/psammead-episode-list/src/index.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import * as scripts from '@bbc/gel-foundations/scripts';
+
+import { renderEpisodes, exampleEpisodes, rtlEpisodes } from './fixtures';
+
+describe('Episode List ', () => {
```

Comment:
Can we have unit tests that are not just snapshots? e.g. checking the correct amount of items are rendered